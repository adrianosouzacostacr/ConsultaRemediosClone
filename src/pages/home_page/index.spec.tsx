import 'react-native';
import React from 'react';
import HomePage from './';

import {render} from '@testing-library/react-native';

describe('Home page', () => {
  it('renders correctly', () => {
    render(<HomePage />);
  });

  it('should display correcly infos', () => {
    const {getAllByA11yLabel} = render(<HomePage />);
    const messageLabel = getAllByA11yLabel('message');
    const brandLabel = getAllByA11yLabel('brand');

    expect(messageLabel[0].children[0]).toEqual('Agora sim vai dar certo \\o/');
    expect(brandLabel[0].children[0]).toEqual('Consulta Rem√©dios');
  });
});
