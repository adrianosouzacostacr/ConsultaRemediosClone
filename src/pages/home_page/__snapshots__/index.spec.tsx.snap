// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page renders correctly 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
      "overflow": "visible",
    }
  }
>
  <Text
    accessibilityLabel="brand"
    style={Object {}}
  >
    Consulta Rem√©dios
  </Text>
  <Text
    accessibilityLabel="message"
    style={Object {}}
  >
    Agora sim vai dar certo \\o/
  </Text>
</RCTSafeAreaView>
`;
